username_api.raml

```diff
@@ -0,0 +1,31 @@
+#%RAML 1.0
+
+title: username-availability API
+version: v1
+protocols: [ HTTPS ]
+baseUri: https://username-availability.herokuapp.com/
+
+types:
+  statusCode:
+    type: number
+    minimum: 100
+    maximum: 599
+    format: int64
+  Check:
+    type: object
+    properties:
+      avatar: null | string
+      possible: boolean
+      status: statusCode
+      url: string
```

Comment:
we've ``profile`` field also in case of ``facebook`` case. also, you added ``usable`` field recently. should they not be included? 

also, some of the fields are not always in the response, in case there is something you need to do for that as well.

Reply:
ok